sources/packages/web/src/services/http/StudentApi.ts

```diff
@@ -135,6 +136,18 @@ export class StudentApi extends HttpBaseClient {
     );
   }
 
+  /**
+   * Get all student documents uploaded by student uploader.
+   * @return list of student documents.
+   */
+  async getInstitutionStudentFiles(
```

Comment:
we already have a api service, just need to adjust the comments and rename
![image](https://user-images.githubusercontent.com/77353155/233170396-0b818c42-0d8f-450f-b5a5-1b1897a61e4f.png)